src/uu/free/src/free.rs

```diff
@@ -90,6 +155,15 @@ fn parse_meminfo() -> Result<MemInfo, Box<dyn std::error::Error>> {
         swap_free: sys.free_swap(),
         swap_used: sys.total_swap().saturating_sub(sys.free_swap()),
         reclaimable: 0,
+        // i dont know how to get that kind of info, you probably have to either pay money to get the info or pay some money for some developer docs
```

Comment:
this document doesn't provide much value, please make it more actionable

Reply:
the only thing i found is https://apple.stackexchange.com/questions/101488/os-xs-free-or-proc-meminfo-equivalent but i dont use macos so i dont actually know where you could get detailed memory information from.